lib/reporters/error-reporter.js

```diff
@@ -0,0 +1,37 @@
+const Logger = require('../logger');
+const colors = require('colors');
+const path = require('path');
+const isArray = require('lodash/isArray');
+
+const logger = new Logger();
+
+function errorReporter(errors, logLevel) {
```

Comment:
why support multiple types for errors? use default parameter value? `errorReporter(errors = []){}`?